components/FeaturedCard.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import {
+  ImageBackground,
+  Image,
+  StyleSheet,
+  TouchableHighlight,
+  View,
+} from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import * as Amplitude from 'expo-analytics-amplitude';
+import { Feather } from '@expo/vector-icons';
+import Colors from '../constants/Colors';
+import { Title, Subtitle } from './shared/Typography';
+import { styles } from './HomeCard';
+
+function FeaturedCard() {
+  const navigation = useNavigation();
+
+  return (
+    <TouchableHighlight
+      underlayColor={Colors.darkBlue}
+      style={[styles.card, { height: 300 }]}
+      onPress={() => {
+        Amplitude.logEventWithProperties('TAP post', { post_type: 'featured' });
+        navigation.navigate('Featured');
+      }}
+    >
+      <View style={cardStyles.mainContainer}>
+        <ImageBackground
+          source={require('../assets/images/hp-bg.jpg')}
```

Comment:
I'm a bit concerned that all of this is static, so to remove it and to update it, we'll need to push new code. Is there anywhere we can fetch this from the inter webs? I'm thinking like a special web page or something of the sort. 

Also, that could be a development opportunity for the future.